core/src/main/java/org/springframework/security/authorization/method/AuthorizationAnnotationUtils.java

```diff
@@ -50,6 +56,41 @@
  */
 final class AuthorizationAnnotationUtils {
 
+	static <A extends Annotation> Function<AnnotatedElement, A> withPlaceholderResolver(Class<A> type) {
+		Function<MergedAnnotation<A>, A> map = (mergedAnnotation) -> {
+			if (mergedAnnotation.getMetaSource() == null) {
+				return mergedAnnotation.synthesize();
+			}
+			PropertyPlaceholderHelper helper = new PropertyPlaceholderHelper("{", "}");
```

Comment:
It might be nice to have a way to allow users to specify `ignoreUnresolvablePlaceholders = false`. For this reason, a boolean for turning this feature on and off might not be the best approach.